packages/shared/src/components/cards/v1/WelcomePostCard.tsx

```diff
@@ -66,35 +80,58 @@ export const WelcomePostCard = forwardRef(function SharePostCard(
         }
       }
     >
-      <PostCardHeader
-        post={post}
-        onMenuClick={(event) => onMenuClick?.(event, post)}
-        metadata={{
-          topLabel: enableSourceHeader ? post.source.name : post.author.name,
-          bottomLabel: enableSourceHeader
-            ? post.author.name
-            : `@${post.source.handle ?? post.sharedPost.source.handle}`,
-        }}
-      >
-        <SquadHeaderPicture
-          author={post.author}
-          source={post.source}
-          reverse={!enableSourceHeader}
-        />
-      </PostCardHeader>
-      <CardTitle
-        className={classNames(
-          generateTitleClamp({
-            hasImage: !!image,
-            hasHtmlContent: !!post.contentHtml,
-          }),
-          'multi-truncate',
-        )}
-      >
-        {post.title}
-      </CardTitle>
+      <CardContainer>
+        <PostCardHeader
+          post={post}
+          onMenuClick={(event) => onMenuClick?.(event, post)}
+          metadata={{
+            topLabel: enableSourceHeader ? post.source.name : post.author.name,
+            bottomLabel: enableSourceHeader
+              ? post.author.name
+              : `@${post.source.handle ?? post.sharedPost.source.handle}`,
+          }}
+        >
+          <SquadHeaderPicture
+            author={post.author}
+            source={post.source}
+            reverse={!enableSourceHeader}
+          />
+        </PostCardHeader>
+
+        <CardContent>
+          <div className="mr-4 flex-1">
+            <CardTitle
+              className={classNames(
+                generateTitleClamp({
+                  hasImage: !!image,
+                  hasHtmlContent: !!post.contentHtml,
+                }),
+                'multi-truncate',
+              )}
+            >
+              {post.title}
```

Comment:
I believe we should be using this here as well  `const { title, summary } = useTruncatedSummary(post);`